<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interfaz</class>
 <widget class="QWidget" name="Interfaz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>633</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="9">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QTextEdit" name="teInfo">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>285</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>285</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="1">
      <widget class="QComboBox" name="listRegiones">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>163</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="listPaises">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>125</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pbCalcular">
       <property name="text">
        <string>Calcular Gráficas</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="lPriFecha">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="11" column="9">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lUltFecha">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="9">
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pbSalir">
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="9">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="label">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Ferreyra Tomás</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="8">
    <widget class="QPushButton" name="pbComparar">
     <property name="minimumSize">
      <size>
       <width>98</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>98</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Comparar Datos</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="1">
      <widget class="QCheckBox" name="chkRecup">
       <property name="text">
        <string>Recuperados</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="chkConfir">
       <property name="text">
        <string>Confirmados</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="chkMuert">
       <property name="text">
        <string>Muertes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="9">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lCero">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="14" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="1">
      <widget class="QProgressBar" name="prbProgreso">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>23</height>
        </size>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pbActualizar">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Actualizar Base de Datos</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lActBD">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>23</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="9" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lTotalCant">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pbSIR">
     <property name="text">
      <string>Calcular SIR</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbSalir</sender>
   <signal>released()</signal>
   <receiver>Interfaz</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>622</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listPaises</sender>
   <signal>highlighted(QString)</signal>
   <receiver>Interfaz</receiver>
   <slot>listar_regiones(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCalcular</sender>
   <signal>pressed()</signal>
   <receiver>Interfaz</receiver>
   <slot>calcular_datos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCalcular</sender>
   <signal>pressed()</signal>
   <receiver>Interfaz</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkConfir</sender>
   <signal>clicked()</signal>
   <receiver>Interfaz</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkRecup</sender>
   <signal>clicked()</signal>
   <receiver>Interfaz</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMuert</sender>
   <signal>clicked()</signal>
   <receiver>Interfaz</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbActualizar</sender>
   <signal>pressed()</signal>
   <receiver>Interfaz</receiver>
   <slot>actualizar_bd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>622</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbComparar</sender>
   <signal>pressed()</signal>
   <receiver>Interfaz</receiver>
   <slot>proximamente()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>622</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Interfaz</sender>
   <signal>sRango(int,int)</signal>
   <receiver>prbProgreso</receiver>
   <slot>setRange(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Interfaz</sender>
   <signal>sValor(int)</signal>
   <receiver>prbProgreso</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSIR</sender>
   <signal>pressed()</signal>
   <receiver>Interfaz</receiver>
   <slot>mostrar_SIR()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>sTexto(QString)</signal>
  <signal>sRango(int,int)</signal>
  <signal>sValor(int)</signal>
  <slot>listar_regiones(QString)</slot>
  <slot>calcular_datos()</slot>
  <slot>actualizar_bd()</slot>
  <slot>proximamente()</slot>
  <slot>mostrar_SIR()</slot>
 </slots>
</ui>
